lucene_solr_nightly_smoketest_OpenJ9 tests failed with `/bin/sh: 7: Syntax error: end of file unexpected (expecting "}")`

```
08:28:55  ===============================================
08:28:55  Running test lucene_solr_nightly_smoketest_OpenJ9_0 ...
08:28:55  ===============================================
08:28:55  lucene_solr_nightly_smoketest_OpenJ9_0 Start Time: Thu Mar  7 07:28:53 2019 Epoch Time (ms): 1551943733382
08:28:55  "/home/jenkins/workspace/openjdk8_j9_externaltest_x86-64_linux/openjdkbinary/j2sdk-image/jre/bin/java" -Xshareclasses:destroyAll; "/home/jenkins/workspace/openjdk8_j9_externaltest_x86-64_linux/openjdkbinary/j2sdk-image/jre/bin/java" -Xshareclasses:groupAccess,destroyAll; echo "cache cleanup done";
08:28:55  JVMSHRC005I No shared class caches available
08:28:55  JVMSHRC005I No shared class caches available
08:28:55  cache cleanup done
08:28:55  variation: NoOptions
08:28:55  JVM_OPTIONS: -Xcompressedrefs 
08:28:55  { itercnt=1; \
08:28:55  mkdir -p "/home/jenkins/workspace/openjdk8_j9_externaltest_x86-64_linux/openjdk-tests/TestConfig/scripts/testKitGen/../../../TestConfig/test_output_15519427955175/lucene_solr_nightly_smoketest_OpenJ9_0"; \
08:28:55  cd "/home/jenkins/workspace/openjdk8_j9_externaltest_x86-64_linux/openjdk-tests/TestConfig/scripts/testKitGen/../../../TestConfig/test_output_15519427955175/lucene_solr_nightly_smoketest_OpenJ9_0"; \
08:28:55  docker run --name lucene-solr-test  adoptopenjdk-lucene-solr-test:latest; \
08:28:55  		 if [ $? -eq 0 ] ; then echo ""; echo "lucene_solr_nightly_smoketest_OpenJ9_0""_PASSED"; echo ""; cd /home/jenkins/workspace/openjdk8_j9_externaltest_x86-64_linux/openjdk-tests/TestConfig/scripts/testKitGen/../../..;  else echo ""; echo "lucene_solr_nightly_smoketest_OpenJ9_0""_FAILED"; echo ""; fi; \
08:28:55  		 docker cp lucene-solr-test:/lucene-solr/lucene/build/core/test "/home/jenkins/workspace/openjdk8_j9_externaltest_x86-64_linux/openjdk-tests/TestConfig/scripts/testKitGen/../../../TestConfig/test_output_15519427955175/lucene_solr_nightly_smoketest_OpenJ9_0"/external_test_reports; \
08:28:55  		 docker rm -f lucene-solr-test
08:28:55  /bin/sh: 7: Syntax error: end of file unexpected (expecting "}")
08:28:55  autoGen.mk:20: recipe for target 'lucene_solr_nightly_smoketest_OpenJ9_0' failed
08:28:55  make[1]: *** [lucene_solr_nightly_smoketest_OpenJ9_0] Error 2
08:28:55  make[1]: Leaving directory '/home/jenkins/workspace/openjdk8_j9_externaltest_x86-64_linux/openjdk-tests/thirdparty_containers/lucene-solr'
08:28:55  /home/jenkins/workspace/openjdk8_j9_externaltest_x86-64_linux/openjdk-tests/TestConfig/scripts/testKitGen/../../../TestConfig/settings.mk:271: recipe for target 'sanity.external-lucene-solr' failed
08:28:55  make: *** [sanity.external-lucene-solr] Error 2
08:28:55  make: Leaving directory '/home/jenkins/workspace/openjdk8_j9_externaltest_x86-64_linux/openjdk-tests/thirdparty_containers'

```

https://ci.adoptopenjdk.net/view/Test_external/job/openjdk8_j9_externaltest_x86-64_linux/362/console

Should be command set issue, missing a comma and a new line.

--- a/thirdparty_containers/lucene-solr/dockerfile/Dockerfile
+++ b/thirdparty_containers/lucene-solr/dockerfile/Dockerfile
@@ -31,7 +31,6 @@ FROM $IMAGE_NAME:$IMAGE_VERSION
 # Install test dependent executable files
 RUN apt-get update \
 	&& apt-get -y install \
-	ant \
 	apt-transport-https \
 	ca-certificates \
 	dirmngr \
@@ -40,11 +39,7 @@ RUN apt-get update \
 	make \
 	unzip \
 	vim \
-	wget \
-	python3-pip python3-dev \
-	&& cd /usr/local/bin \
-	&& ln -s /usr/bin/python3 python \
-	&& pip3 install --upgrade pip 
+	wget
 
 WORKDIR /
 ENV LUCENE_SOLR_HOME ${WORKDIR}
@@ -54,7 +49,7 @@ ENV ANT_VERSION 1.10.3
 RUN wget --no-check-certificate --no-cookies https://archive.apache.org/dist/ant/binaries/apache-ant-${ANT_VERSION}-bin.tar.gz && \
     gunzip apache-ant-${ANT_VERSION}-bin.tar.gz && \
     tar -xvf apache-ant-${ANT_VERSION}-bin.tar
-	
+
 #apache ivy setup 
 ENV IVY_VERSION 2.4.0
 RUN wget --no-check-certificate --no-cookies https://archive.apache.org/dist/ant/ivy/${IVY_VERSION}/apache-ivy-${IVY_VERSION}-bin.tar.gz && \

--- a/thirdparty_containers/lucene-solr/dockerfile/lucene-solr-test.sh
+++ b/thirdparty_containers/lucene-solr/dockerfile/lucene-solr-test.sh
@@ -17,13 +17,11 @@ if [ -d /java/jre/bin ];then
 	export JAVA_BIN=/java/jre/bin
 	export JAVA_HOME=/java
 	export PATH=$JAVA_BIN:$PATH
-	java -version
 elif [ -d /java/bin ]; then
 	echo "Using mounted Java9"
 	export JAVA_BIN=/java/bin
 	export JAVA_HOME=/java
 	export PATH=$JAVA_BIN:$PATH
-	java -version
 else
 	echo "Using docker image default Java"
 	java_path=$(type -p java)
@@ -31,18 +29,9 @@ else
 	java_root=${java_path%$suffix}
 	export JAVA_BIN="$java_root"
 	echo "JAVA_BIN is: $JAVA_BIN"
-	$JAVA_BIN/java -version
 	export JAVA_HOME="${java_root%/bin}"
 fi
 
-TEST_SUITE=$1
-
-echo "PATH is : $PATH"
-echo "JAVA_HOME is : $JAVA_HOME"
-echo "ANT_HOME is: $ANT_HOME" 
-echo "type -p java is :"
-type -p java
-echo "java -version is: \n"
 java -version
 cd ${LUCENE_SOLR_HOME}/lucene-solr
 ls .

--- a/thirdparty_containers/lucene-solr/playlist.xml
+++ b/thirdparty_containers/lucene-solr/playlist.xml
@@ -38,7 +38,7 @@
 		<command>docker run --name lucene-solr-test $(EXTRA_DOCKER_ARGS) adoptopenjdk-lucene-solr-test:latest; \
 			 $(TEST_STATUS); \
 			 docker cp lucene-solr-test:/lucene-solr/lucene/build/core/test $(REPORTDIR)/external_test_reports; \
-			 docker rm -f lucene-solr-test
+			 docker rm -f lucene-solr-test; \
 			 docker rmi -f adoptopenjdk-lucene-solr-test
 		</command>
 		<levels>

